crates/node-bindings/src/parcel/parcel.rs

```diff
@@ -41,6 +45,19 @@ impl ParcelNapi {
     })
   }
 
+  #[napi]
+  pub async fn build(&self) -> napi::Result<()> {
+    self.parcel.build().unwrap();
+    Ok(())
+  }
```

Comment:
Initially this will return an `AssetGraph` which will plug into the existing bundler